opencsp/common/lib/render_control/RenderControlTower.py

```diff
@@ -0,0 +1,47 @@
+import opencsp.common.lib.render_control.RenderControlPointSeq as rcps
+import opencsp.common.lib.render_control.RenderControlText as rctxt
+from opencsp.common.lib.render_control.RenderControlPointSeq import RenderControlPointSeq
+
+
+class RenderControlTower:
+    """
+    Render control for towers.
+    """
+
+    def __init__(
+        self,
+        centroid: bool = False,
+        draw_name=False,
+        name_style=rctxt.default(color='k'),
+        draw_outline=True,
+        point_styles: RenderControlPointSeq = rcps.marker(),
+        wire_frame: RenderControlPointSeq = rcps.outline(),
+        target: RenderControlPointSeq = rcps.marker(marker='x', color='r', markersize=6),
+    ) -> None:
+
```

Comment:
Please add a docstring describing the input parameters.

Reply:
done!